### PR TITLE
Add zoom buttons to the location view

### PR DESCRIPTION
**Pull request by andybalaam**
zooming

---
Here's what your changelog entry will look like:

## ✨ Features
 * Add zoom buttons to the location view ([\#7482](https://github.com/matrix-org/matrix-react-sdk/pull/7482)). Contributed by @andybalaam.

Preview: https://61d867651f962533d6059167--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

**Comment by andybalaam** [comment]
I am chatting with our designers about whether the bottom right is a little cluttered with the copyright message/button as well as the +/- buttons.

File: res/css/views/dialogs/_LocationViewDialog.scss [hunk 1]
```diff
@@ -49,8 +49,48 @@ limitations under the License.
...
 
-    .mx_MLocationBody .mx_MLocationBody_map {
-        width: 80vw;
-        height: 80vh;
+    .mx_MLocationBody {
+        position: absolute;
+
+        .mx_MLocationBody_map {
+            width: 80vw;
+            height: 80vh;
```
Comment (turt2live) [inline res/css/views/dialogs/_LocationViewDialog.scss:57]: viewport sizing is something we don't typically use - if it works, it works, but just a heads up.

**Review by turt2live — APPROVED** [review]
# Zoom

<sup>zoom</sup>

File: res/css/views/dialogs/_LocationViewDialog.scss (hunk 1 above)
Comment (andybalaam) [reply res/css/views/dialogs/_LocationViewDialog.scss:57]: Thank yeah, I noticed that, but it was the only way I could make it work in this instance. It took quite a lot of fiddling to get this to look right.

**Comment by gaelledel** [comment]
@andybalaam It's fine you can leave the controls at the bottom right

**Pull request merged**